README.md

```diff
@@ -99,6 +99,46 @@ config.settings.DSGVOBanner = {
   privacy_url: '/privacy',
 };
 ```
+You can customize the Style of the Modal Buttons
+```
+config.settings.DSGVOBanner = {
+  bannerAgreeButtonColor: "branded olive",
+  bannerAdjustButtonColor: "branded blue",
```

Comment:
I would put these settings together in one object, and avoid using the word "color", since it's really configuring CSS classes that can control more styles than just the color.

```
config.settings.DSGVOBanner = {
  ...(config.settings.DSGVOBanner || {}),
  cssClasses: {
    bannerAgreeButton: 'branded olive',
    bannerAdjustButton: 'branded blue'
  }
}
```